Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM alpine:latest as alpine
```

Comment:
Is it possible to name the separate build as another name? Like `as extractwar` for more clarification on what is it really doing?